docs/fsharp/language-reference/generics/statically-resolved-type-parameters.md

```diff
@@ -61,6 +60,32 @@ The output is as follows.
 1.500000
 ```
 
+Starting with F# 4.1, you can also specify concrete type names in statically resolved type parameter signatures.  In previous versions of the language, the type name could actually be inferred by the compiler, but could not actually be specified in the signature.  This quirk in the language is addressed in F# 4.1, but is also backwards-compatible.  Here's an example:
```

Comment:
Good point.  I did prune some of the older references from the docs when I did a round cleanups a while back.  I suppose I or someone else would do that for these.